File: src/main/java/com/formssihk/cordaexample/api/payload/CreatePayload.java
package com.formssihk.cordaexample.api.payload;

import lombok.Data;

import java.time.Instant;
import java.util.List;
import java.util.Map;
@Data
public class CreatePayload {
    String userPublicKey;

    Map<String, Instant> loanRecords;

}

File: src/main/kotlin/com/formssihk/cordaexample/api/payload/CreatePayload.kt
package com.formssihk.cordaexample.api.payload

import com.formssihk.sample.payload.Loan
import net.corda.core.contracts.StateRef
import java.time.Instant

data class CreatePayload (
    var userPublicKey: String,
    var loadRecords: Map<String, Instant>?
    ) {
}

data class UpdatePayload (
    val userPublicKey: String,
    val loadRecords: List<Loan>
    )